src/main/java/seedu/exercise/model/regime/UniqueRegimeList.java

```diff
@@ -1,141 +1,18 @@
 package seedu.exercise.model.regime;
 
-import static java.util.Objects.requireNonNull;
-import static seedu.exercise.commons.util.CollectionUtil.requireAllNonNull;
-
-import java.util.Iterator;
-import java.util.List;
-
-import javafx.collections.FXCollections;
-import javafx.collections.ObservableList;
-import seedu.exercise.model.regime.exceptions.DuplicateRegimeException;
-import seedu.exercise.model.regime.exceptions.RegimeNotFoundException;
+import seedu.exercise.model.Resource;
+import seedu.exercise.model.UniqueList;
 
 /**
  * A list of regimes that enforces uniqueness between its elements and does not allow nulls.
  * A regime is considered unique by comparing using {@code Regime#isSameRegime(Regime)}.
  * As such, adding and updating of regimes uses Regime#isSameRegime(Regime) for equality so as to ensure that
  * the regime being added or updated is unique in terms of identity in the UniqueRegimeList.
- *
+ * <p>
```

Comment:
closing tags. \</p>

Reply:
File removed too with reasons similar to ``UniqueExerciseList``.